crypten/encoder.py

```diff
@@ -64,7 +64,7 @@ def decode(self, tensor):
         assert is_int_tensor(tensor), "input must be a LongTensor"
         if self._scale > 1:
             correction = (tensor < 0).long()
-            dividend = tensor / self._scale - correction
+            dividend = tensor // self._scale - correction
```

Comment:
Thanks for noticing this. The definitions of truediv has been changed recently in PyTorch for the torch.long dtype. The previous behavior was identical to floordiv.